CASMNET-1746 | adding network troubleshoooting documentation and re-organizing content (#2145)
Cray-HPE/docs-csm: troubleshooting/README.md

This document provides links to troubleshooting information for services and functionality provided by CSM.

> ## Helpful tips navigating around the CSM repository:

Reviewer: ```suggestion
## Helpful tips for navigating the CSM repository
```

> In the main repository landing page:

Reviewer: ```suggestion
In the main repository landing page, change the branch to the CSM version being used on the system (for example, release/1.0, release/1.2, release/1.3).
```

> 

Reviewer: ```suggestion
```

> * Change the branch to the CSM version being used on the system (e.g., release/1.0, release/1.2, release/1.3

Reviewer: ```suggestion
```

> Use the pre-populated GitHub “Search or jump to...” function in the upper left hand of the page and append keywords related to the exiting problem seen into the existing search. (The example searches for “ping” and “PXE” related troubleshooting resources on the “main” branch.)

Reviewer: ```suggestion
Use the pre-populated GitHub “Search or jump to...” function in the upper left hand side of the page and append keywords related to the exiting problem seen into the existing search. (The example searches for “ping” and “PXE” related troubleshooting resources on the “main” branch.)
```

> * Follow any run-books, guides or procedures which are directly related to the problem.

Reviewer: ```suggestion
* Follow any run-books, guides, or procedures which are directly related to the problem.
```

* Change the branch to “main” and search a second time to retrieve very recent or beta run-books and guides.

> * The user could also expand the search beyond the “troubleshooting” section (instead of doing “path troubleshooting”) and/or use more advanced GitHub searches such as "path configure" to find the right context.

Reviewer: ```suggestion
* Users can also expand the search beyond the “troubleshooting” section (instead of doing “path troubleshooting”) and/or use more advanced GitHub searches such as "path configure" to find the right context.
```

## Quick links:

* [Known issues](#known-issues)
* [Booting](#booting)

> * [ConMan](#conman)

Author: let me push a change and change the order. thank you

> * [ConMan](#conman)

Author: done
